src/app/zap-templates/templates/chip/helper.js

```diff
@@ -319,18 +335,19 @@ function chip_available_cluster_commands(options)
 //
 // Module exports
 //
-exports.chip_clusters                   = chip_clusters;
-exports.chip_has_clusters               = chip_has_clusters;
-exports.chip_client_clusters            = chip_client_clusters;
-exports.chip_has_client_clusters        = chip_has_client_clusters;
-exports.chip_server_clusters            = chip_server_clusters;
-exports.chip_has_server_clusters        = chip_has_server_clusters;
-exports.chip_cluster_commands           = chip_cluster_commands;
-exports.chip_cluster_command_arguments  = chip_cluster_command_arguments;
-exports.chip_server_global_responses    = chip_server_global_responses;
-exports.chip_cluster_responses          = chip_cluster_responses;
-exports.chip_cluster_response_arguments = chip_cluster_response_arguments
-exports.chip_attribute_list_entryTypes  = chip_attribute_list_entryTypes;
-exports.chip_server_cluster_attributes  = chip_server_cluster_attributes;
-exports.chip_server_has_list_attributes = chip_server_has_list_attributes;
-exports.chip_available_cluster_commands = chip_available_cluster_commands;
+exports.chip_clusters                     = chip_clusters;
+exports.chip_has_clusters                 = chip_has_clusters;
+exports.chip_client_clusters              = chip_client_clusters;
+exports.chip_has_client_clusters          = chip_has_client_clusters;
+exports.chip_server_clusters              = chip_server_clusters;
+exports.chip_has_server_clusters          = chip_has_server_clusters;
+exports.chip_cluster_commands             = chip_cluster_commands;
+exports.chip_cluster_command_arguments    = chip_cluster_command_arguments;
+exports.chip_server_global_responses      = chip_server_global_responses;
+exports.chip_cluster_responses            = chip_cluster_responses;
+exports.chip_cluster_response_arguments   = chip_cluster_response_arguments
+exports.chip_attribute_list_entryTypes    = chip_attribute_list_entryTypes;
+exports.chip_server_cluster_attributes    = chip_server_cluster_attributes;
+exports.chip_server_has_list_attributes   = chip_server_has_list_attributes;
+exports.chip_available_cluster_commands   = chip_available_cluster_commands;
+exports.chip_server_has_struct_attributes = chip_server_has_struct_attributes;
```

Comment:
nit: Move it one line upper so it lives next to `chip_server_has_list_attributes`